styles/themes/common/_index.scss

```diff
@@ -14,4 +14,23 @@
     --g-border-radius-xl: 10px;
 
     --g-focus-border-radius: 2px;
+
+    @at-root body#{&} {
```

Comment:
The root can be on any element, not only the body. Also these styles should go into styles.scss, not here

Reply:
These defaults only for the class on the body, if present, any other element always has `dir` attribute if it set to `ThemeProvider`.